sv_inline.h

```diff
@@ -68,13 +68,13 @@
 SV* Perl_more_sv(pTHX);
 
 /* new_SV(): return a new, empty SV head */
-
-#ifdef DEBUG_LEAKING_SCALARS
-/* provide a real function for a debugger to play with */
-STATIC SV*
-S_new_SV(pTHX_ const char *file, int line, const char *func)
+PERL_STATIC_INLINE SV*
+Perl_new_SV(pTHX_ const char *file, int line, const char *func)
 {
     SV* sv;
+    PERL_UNUSED_ARG(file);
+    PERL_UNUSED_ARG(line);
+    PERL_UNUSED_ARG(func);
```

Comment:
These three variables are used when DEBUG_LEAKING_SCALARS is defined.  Maybe these should be wrapped with a `#ifndef DEBUG_LEAKING_SCALARS ... #endif`.

Reply:
Sure, I can do that.